content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,191 @@
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p18](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+In other words, foods containing soy can have very unpredictable amounts of phytoestrogens, processed foods can lose a lot of their isoflavone content, and usually the effect of cooking soy based foods is to reduce the amount of isoflavones dramatically.
+
+
+### 2.2 How much is safe to eat is not well understood.
+
+While there is no RDA currently published by any body we've been able to find, there are some indications of what might constitute a dosage which would have an effect on the body.  
+
+> At the same time, to have the beneficial effects of soy isoflavones, intake should be at least of 60-100 mg per day, at present not easily reached in Western countries.  
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p21](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+This talks specifically about beneficial effects (which has been the focus of a lot of research due to health claims for tofu in the 1990s), the full scope of large dosages of phytoestrogens (whatever may constitute large dosages!) can have negative effects.
+
+> There are also real concerns that excessive amounts of isoflavones in serum may promote other hormone-related problems.  
+> - [An updated review of dietary isoflavones: Nutrition, processing, bioavailability and impacts on human health - Critical Reviews in Food Scient and Nutrition - 2017](http://web.archive.org/web/20191015205851/https://www.researchgate.net/profile/Humayoun_Akhtar/publication/283791348_An_Updated_Review_of_Dietary_Isoflavones_Nutrition_Processing_Bioavailability_and_Impacts_on_Human_Health/links/5a2f732a4585155b617a45fd/An-Updated-Review-of-Dietary-Isoflavones-Nutrition-Processing-Bioavailability-and-Impacts-on-Human-Health.pdf?origin=publication_detail)
+
+and 
+
+> [P]hytoestrogens are also considered endocrine disruptors, indicating that they have the potential to cause adverse health effects such as infertility and increased risks of cancer in oestrogen-sensitive organs.  
+> - [The potential health effects of dietary phytoestrogens - British Journal of Pharmacology - 2017](https://web.archive.org/web/20190922065924/https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1111/bph.13622)  
+
+That said, the few studies that show negative effects have major drawbacks such as:
+
+1. Being performed on animals, which apart from the ethical concerns do not even necessarily yield valid results for humans! 
+2. Being performed in-vitro (i.e. in a petri-dish on small clusters of cells)
+3. Drawing conclusions from statistical (e.g. epidiemological) data which risks confusing mere correlation with cause.
```

Comment:
Could (for 1.) possible add something about soy absorption methods in rats differing wildly from humans. Though its not really necessary

Reply:
It's already quite a long sentence so I'm going to skip it! I was originally going to dive into more detail as to why these various studies are not good data, but like you say, I don't think it's necessary.